### ISSUE TITLE
packages are not uploaded as artifacts

### ISSUE THREAD

**Issue by Borda** [Lightning-AI/utilities#66]
## 🐛 Bug

After making release packages are not uploaded to Release artifacts

### To Reproduce

https://github.com/Lightning-AI/utilities/actions/runs/3362708729/jobs/5574801044
```
Warning: Cannot upload assets for release which is being undefined
```

**Comment by shenoynikhil** [comment]
Do you think adding [this reference tag](https://github.com/AButler/upload-release-assets/issues/6#issuecomment-1304355941) could be a feasible solution?

**Comment by Borda** [comment]
> Do you think adding [this release tag](https://github.com/AButler/upload-release-assets/issues/6#issuecomment-1304355941) could be a feasible solution?

We can try or just drop the tag event...

**Comment by Borda** [comment]
I believe it was fixed by face8f06c7dd81db2991e2f8e815e0b300340081
